tests/jenkins/TestDataPrepperDistributionArtifacts.groovy

```diff
@@ -3,35 +3,68 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import jenkins.tests.BaseJobPipelineTest
+
+import jenkins.tests.BuildPipelineTest
 import org.junit.Before
 import org.junit.Test
 
-class TestDataPrepperDistributionArtifacts extends BaseJobPipelineTest {
+class TestDataPrepperDistributionArtifacts extends BuildPipelineTest {
```

Comment:
I think you should still inherit from `BaseJobPipelineTest` which sets up library and scm, or why aren't those needed?

Reply:
We won't be able to mock our libraries with `BuildPipelineTest `.
I want to remove `BaseJobPipelineTest` because it would allow the developer to mock our libraries.